Dockerfile

```diff
@@ -8,14 +8,15 @@ FROM ${BASE_IMAGE_NAME}:${BASE_IMAGE_TAG}
 # ... then the modules installed during the transformation are not loaded automatically!
 # ... because the loader thinks that this directory does not exist (it did not exist at the start of the script)
 # Eg. mkdir -p /var/www/.local/lib/python3.8/site-packages
-RUN mkdir -p $(su www-data -s /bin/bash -c "python -c 'import site; print(site.USER_SITE)'")
+RUN . $VIRTUAL_ENV"/bin/activate" && mkdir -p $(su www-data -s /bin/bash -c "python -c 'import site; print(site.USER_SITE)'")
```

Comment:
```suggestion
RUN . "$VIRTUAL_ENV/bin/activate" && mkdir -p $(su www-data -s /bin/bash -c "python -c 'import site; print(site.USER_SITE)'")
```

Reply:
You should always put path inside quotes. Otherwise it the path contains a space, Bash will think it's other argument.